### PR TITLE
update init system and default manifest templates

### PR DESCRIPTION
**Pull request by ddollar**
## Release Playbook
- [x] Rebase against master
- [x] Release branch (20160513200733-init)
- [x] Pass CI (https://circleci.com/gh/convox/rack/762)
- [x] Code review
- [ ] Merge into master
- [ ] Release master ()
- [ ] Pass CI ()
- [ ] Update staging rack
- [ ] Edit [Rack release record](https://github.com/convox/rack/releases) and/or update [docs](https://github.com/convox/convox.github.io)
- [ ] Publish release
- [ ] Release CLI

### CODE REVIEW

File: cmd/convox/templates/init/ruby/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM convox/ruby
+
+# copy only the files needed for bundle install
+# uncomment the vendor/cache line if you `bundle package` your gems
```
Comment (nzoschke) [inline cmd/convox/templates/init/ruby/Dockerfile]: is `bundle package` recommended?

Comment (ddollar) [inline cmd/convox/templates/init/ruby/Dockerfile]: Not sure yet. It doesn't seem to make much of a speed difference but it does drop some internet-dependency. Unfortunately it's hard to conditionally copy a directory if it exists.

**Comment by nzoschke** [comment]
If you already know Alpine wont cut it should we not ship that?

If I had to make a snap judgement, I'd say Ubuntu 14.04 still has a lot of advantages here:

* It's a good "consumer" distro in general with tons of prior art for running apps
* ElasticBeanstalk and Heroku use it for Rails
* Canonical will support the OS until April 2019
* Docker builds and supports a base image that is 188 MB

That said I'm also OK shipping an Alpine based one as I do want to support that in general too.

**Comment by nzoschke** [comment]
I am testing this on top of the #593 and local SSL is not working.

```
$ docker ps
CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                                           NAMES
356335e8cb3b        convox/proxy        "proxy 443 172.17.0.1"   42 seconds ago      Up 41 seconds       0.0.0.0:443->443/tcp                            adoring_albattani
2c887700bc63        hello/web           "bin/web"                52 seconds ago      Up 51 seconds       0.0.0.0:80->4000/tcp, 0.0.0.0:59540->4001/tcp   hello-web
```

**Comment by ddollar** [comment]
I think local SSL is related to convox proxy on docker for Mac

**Comment by ddollar** [comment]
Have also repackaged the base images in terms of ubuntu 16.04

**Comment by mattmanning** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;6dPhyiaG&#x2F;62-rails-example-app"> rails example app</a>

**Comment by prognostikos** [comment]
@ddollar can you rebase this to latest master? Will be a great help for #628

**Comment by ddollar** [comment]
Rebased, tests are running now

**Pull request merged**